AGENTS.md

```
@@ -1,649 +1,235 @@
-5. Serves a React dashboard for administration, monitoring, and company graph management
-6. Provides Langfuse observability, Telegram bot control, and GitHub integration
-
-**Production URL:** `https://local-llm-server.strikersam.workers.dev`
-**Primary Owner:** strikersam@gmail.com
-
----
-
-## Architecture Overview
+## Architecture
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)